server/utils/jwt.go

```diff
@@ -0,0 +1,43 @@
+package utils
+
+import (
+	"errors"
+	"net/http"
+	"os"
+
+	"github.com/codeday-labs/bookworms/server/db"
+	"github.com/golang-jwt/jwt"
+)
+
+type Claims struct {
+	User db.User
+	jwt.StandardClaims
+}
+
+var Jwt_key = []byte(os.Getenv("kdksksm2039"))
+
+func GetUserFromCookie(r *http.Request) (*db.User, error) {
```

Comment:
Looks like token expiration case is not covered.

Reply:
Validation for that is done by the library internally, you can check line 30 
I have also done a test for that where i set up a to last for 30 minutes and it threw an error

Reply:
Got it, thanks!